etc/tools/.rat-excludes

```diff
@@ -0,0 +1,33 @@
+target
+MANIFEST.in
+.gitignore
+.gitattributes
+.project
+.classpath
+.rat-excludes
+.*ipynb
+.ipynb_checkpoints/
+.*md
+.*avsc
+.*txt
+.*json
+.*data
+.*log
+.*out
+.*parquet
+.*csv
+.*tsv
+.*iml
+service.properties
+db.lck
+DESCRIPTION
+NAMESPACE
+package-sparkR.sh
+\.lintr
+.*ivy.xml
+\.example-image
+\.ensime
+\.example-image
+\.jvmopts
+\.pairs
+\.travis.yml
```

Comment:
This list is to broad, and it should only contain files that does not support adding the apache license header As an example, see how travis.yml does support it : https://github.com/apache/incubator-zeppelin/blob/master/.travis.yml